src/SamplesApp/SamplesApp.UITests.Generator/Uno.Samples.UITest.Generator.csproj

```diff
@@ -1,16 +1,32 @@
-<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
 	<PropertyGroup>
 		<TargetFramework>netstandard2.0</TargetFramework>
 		<DebugType>portable</DebugType>
 		<DebugSymbols>true</DebugSymbols>
 		<AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+
+		<!-- Required to get roslyn workspaces files to be present -->
+		<CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
 	</PropertyGroup>
+	<ItemGroup>
+	  <Compile Include="..\..\SourceGenerators\Uno.UI.SourceGenerators\Helpers\DependenciesInitializer.cs" Link="Helper\DependenciesInitializer.cs" />
+	</ItemGroup>
 	
 	<ItemGroup>
 		<PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0" PrivateAssets="All" />
 		<PackageReference Include="Uno.RoslynHelpers" Version="1.2.0-dev.10" />
-		<PackageReference Include="Microsoft.CodeAnalysis.Workspaces.Common" Version="1.3.2" />
-		<PackageReference Include="Uno.SourceGeneration" />
+		<PackageReference Include="Microsoft.CodeAnalysis">
+
+			<!--
+			This version needs to be aligned with the CI build override MicrosoftNetCompilerVersionOverride, as long as
+			SnapShotTestGenerator uses an inner Roslyn Workspace to generate the snapshots test.
+			-->
+			<Version>4.0.1</Version>
+			
+			<!-- Workspaces need to be included (and without ExcludeAssets) to allow for SnapShotTestGenerator to run properly -->
```

Comment:
I'm confused by this comment. The reference for `Microsoft.CodeAnalysis.Workspaces` was actually removed (which is definitely a good move), but the comment is saying it needs to be included.